cime_config/buildnml

```diff
@@ -149,15 +149,15 @@ def buildnml(case, caseroot, compname):
 
     #HK use C. Bitz grids
     #file1 = os.path.join(din_loc_root, "wav", "ww3", "core2_G4_wns_30min_20000601_to_05.nc")
-    file1 = os.path.join("/glade/scratch/hkershaw/wwatch3v6p07/IC4M3IS0k_gx17/", "wind.ww3")
+    file1 = os.path.join("/glade/scratch/hkershaw/wwatch3v6p07/IC4M3IS0k_gx17_steps180-180-180-15/", "wind.ww3")
     #file1 = os.path.join("/users/hkershaw/data/hkershaw/nuopc/IC4M3IS0k", "wind.ww3")
     file2 = os.path.join(rundir, "wind.ww3")
     if os.path.isfile(file1):
         shutil.copy(file1, file2)
 
     #HK use C. Bitz grids
     #file1 = os.path.join(din_loc_root, "wav", "ww3", "G4L1.mod_def.ww3.121031")
-    file1 = os.path.join("/glade/scratch/hkershaw/wwatch3v6p07/IC4M3IS0k_gx17/", "mod_def.ww3")
+    file1 = os.path.join("/glade/scratch/hkershaw/wwatch3v6p07/IC4M3IS0k_gx17_steps180-180-180-15/", "mod_def.ww3")
```

Comment:
This restart has the timestep matching the code. 
May not need this, I think you can have the timestep different from the restart.